packaging/hudi-utilities-bundle/pom.xml

```diff
@@ -133,6 +133,7 @@
                   <include>io.confluent:common-config</include>
                   <include>io.confluent:common-utils</include>
                   <include>io.confluent:kafka-schema-registry-client</include>
+                  <include>io.confluent:kafka-protobuf-serializer</include>
```

Comment:
Should we also include this in `hudi-integ-test-bundle`?

Reply:
I am not sure. What is the criteria for including it in the integ test bundle?

Reply:
I think if we're not using it in the integ tests, we should not add it.